src/frontend/komponenter/Behandling/Blankett/Blankett.tsx

```diff
@@ -51,14 +54,34 @@ const Blankett: React.FC<Props> = ({ behandlingId }) => {
         });
     };
 
+    const erBehandlingÅpen = (status: BehandlingStatus): boolean => {
+        if (
+            status ===
+            (BehandlingStatus.FATTER_VEDTAK ||
+                BehandlingStatus.IVERKSETTER_VEDTAK ||
+                BehandlingStatus.FERDIGSTILT)
+        ) {
+            return false;
+        }
+        return true;
+    };
+
     return (
         <>
-            <StyledBlankett>
-                <GenererBlankett onClick={genererBlankett}>Generer blankett</GenererBlankett>
-                <HentBlankett onClick={hentBlankett}>Hent blankett</HentBlankett>
-                <PdfVisning pdfFilInnhold={blankettRessurs}></PdfVisning>
-            </StyledBlankett>
-            <BlankettFooter behandlingId={behandlingId} />
+            <DataViewer response={{ behandling }}>
+                <StyledBlankett>
+                    {erBehandlingÅpen(behandling.data.status) && (
+                        <GenererBlankett onClick={genererBlankett}>
+                            Generer blankett
+                        </GenererBlankett>
+                    )}
+                    <HentBlankett onClick={hentBlankett}>Hent blankett</HentBlankett>
+                    <PdfVisning pdfFilInnhold={blankettRessurs}></PdfVisning>
+                </StyledBlankett>
+                {erBehandlingÅpen(behandling.data.status) && (
+                    <BlankettFooter behandlingId={behandlingId} />
+                )}
+            </DataViewer>
```

Comment:
Det DataViewer gjør er å pakke ut data fra Ressurs, slik att dataviewer sjekker for `status === SUKSESS` og så returnerer den det utpakkede data objektet, slik att man ønsker å rendere children med en funksjon
`{({behandling}) = > <>...</>}` 
```suggestion
        <DataViewer response={{ behandling }}>
            {({ behandling }) => (
                <>
                    <StyledBlankett>
                        {erBehandlingÅpen(behandling.status) && (
                            <GenererBlankett onClick={genererBlankett}>
                                Generer blankett
                            </GenererBlankett>
                        )}
                        <HentBlankett onClick={hentBlankett}>Hent blankett</HentBlankett>
                        <PdfVisning pdfFilInnhold={blankettRessurs}></PdfVisning>
                    </StyledBlankett>
                    {erBehandlingÅpen(behandling.status) && (
                        <BlankettFooter behandlingId={behandlingId} />
                    )}
                </>
            )}
        </DataViewer>
```

Reply:
Akkurat, så det er sånn man "pakker ut" typen fra Ressurs 👍